types/react-loadable-visibility/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "jsx": "react",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true
+    },
+    "files": [
+        "index.d.ts",
+        "loadable-components.d.ts",
+        "react-loadable.d.ts",
+        "react-loadable-visibility-tests.tsx"
+    ],
+    "paths": {
```

Comment:
got it. this should be inside `"compilerOptions"` for it to work. Not sure why we don't give an error for this.

Reply:
Done. I have also created a PR to improve the error message that lead me here:
https://github.com/microsoft/types-publisher/pull/653